packages/server-core/src/analytics/analytics/analytics.hooks.ts

```diff
@@ -41,6 +48,70 @@ import {
   analyticsResolver
 } from './analytics.resolvers'
 
+function checkQueryAction(action: 'dailyUsers' | 'dailyNewUsers') {
```

Comment:
We don need this hook as we already have one at `packages/server-core/src/hooks/is-action.ts`

Reply:
removed!